src/curriculum.ts

```diff
@@ -1,29 +1,42 @@
-interface Experiencia{
-  fechaInicio: Date;
-  fechaFin: Date;
-  Puesto: string;
+enum puestoTrabajo{
+  Programador = "Programador",
+  DesarrolladorSoftware = "Desarrollador Software",
+  IngenieroDevOps = "Ingeniero DevOps",
+  ExpertoSeguridad = "Experto Seguridad Informática",
+  ExpertoRedes = "Experto en Redes",
+  DataAnalist = "Data Analist"
 }
 
-interface Estudio{
-  fechaInicio: Date;
-  fechaFin: Date;
-  estudio: string;
+enum nivelIdioma{
+  A1 = "A1",
+  A2 = "A2",
+  B1 = "B1",
+  B2 = "B2",
+  C1 = "C1",
+  C2 = "C2"
 }
 
-interface Idioma{
-  idioma : string;
-  nivel : string;
+type Experiencia={
```

Comment:
¿Este "type" cómo tienes pensado que se use en la lógica de negocio?

Reply:
Como describo en #22 , la experiencia de un candidato se representará como un par de tiempo trabajado y puesto de trabajo. A la hora de usar la lógica de negocio se podrán eliminar los curriculums que no cumplan el tiempo trabajado en ese puesto y a la hora de buscar el mejor candidato se podrá buscar el que más experiencia tenga en un puesto.

Reply:
He estado mirando [aquí](https://blog.logrocket.com/types-vs-interfaces-in-typescript/) las diferencias que existen entre usar un _type_ y un _interface_ en typescript ya que antes lo habías implementado con _interface_. Me parece que ha sido mejor decisión terminar usando un _type_ para almacenar los datos de la experiencia ya que no voy a necesitar funciones especificas en ese tipo. :ok_hand: